key/src/lib.rs

```diff
@@ -26,52 +26,41 @@ extern crate rand_xorshift;
 extern crate rlp;
 extern crate rustc_hex;
 extern crate rustc_serialize;
-extern crate secp256k1;
 extern crate serde;
 #[macro_use]
 extern crate serde_derive;
 extern crate serde_json;
+extern crate sodiumoxide;
 
 mod address;
-mod ecdsa;
+mod ed25519;
+mod ed25519_private;
```

Comment:
How about move `ed25519_private` and`ed25519_public` under the `ed25519` module?

Reply:
I made `ed25519` module and put them into it.